Realm.xcodeproj/project.pbxproj

```diff
@@ -3459,6 +3481,7 @@
 			isa = XCBuildConfiguration;
 			baseConfigurationReference = 5DD755E01BE05C19002800DA /* Tests.xcconfig */;
 			buildSettings = {
+				ALWAYS_EMBED_SWIFT_STANDARD_LIBRARIES = YES;
```

Comment:
This seems out of place? @tgoyne

Reply:
There shouldn't be any reason that this is needed.